### ISSUE TITLE
Installing on Mac OS X need to check for existence of /usr/local/bin

### ISSUE THREAD

**Issue by gtfierro** [immesys/bw2#21]
If user hasn't done anything with Homebrew before (or other package managers), then the usual `curl get.bw2.io/agent | sh' install will fail because it assumes existence of `/usr/local/bin`. 

Fix: something like

```bash
#!/bin/sh
if [ ! -e /usr/local/bin ]; then
   mkdir -p /usr/local/bin
fi
```

**Comment by immesys** [comment]
doesn't mkdir -p just work even if it exists?

**Comment by gtfierro** [comment]
Yeah, I guess it would, but I thought it might give an error if it already existed and if you had `set -ex` on, then I thought I would save the trouble of trying to do directory detection

**Comment by immesys** [comment]
`-p` means no error if it exists along with the parent thing.

```
michael@shadowfall:~$ mkdir -p /tmp
michael@shadowfall:~$ echo $?
0
```

I'll add the mkdir in the next release
